assets/airports/kmsy.json

```diff
@@ -0,0 +1,2833 @@
+    "ctr_radius": 70,
+    "ctr_ceiling": 15000,
+    "initial_alt": 4000,
+    "position": [29.9932778,-90.2590278, "3ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": [29.9932778,-90.2590278],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 60,
+        "speed": 5
+    },
+    "arrivalRunway": "11",
+    "departureRunway": "2",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 150,
+            "airspace_class": "B",
+            "poly": [
+                ["N30.00003", "W90.83335"],
+                ["N30.21668", "W90.88337"],
+                ["N30.43111", "W90.73222"],
+                ["N30.46750", "W89.87778"],
+                ["N30.38333", "W89.61667"],
+                ["N30.35002", "W89.61670"],
+                ["N30.00310", "W89.50191"],
+                ["N29.80838", "W89.50001"],
+                ["N29.45000", "W89.83333"],
+                ["N29.31667", "W90.01528"],
+                ["N29.31723", "W90.16718"],
+                ["N29.31671", "W90.41142"],
+                ["N29.31675", "W90.50085"],
+                ["N29.31671", "W90.52198"],
+                ["N29.31667", "W90.54279"],
+                ["N29.53338", "W90.54164"]
+            ]
+        },
+        {
+            "floor": 0,
+            "ceiling": 110,
+            "airspace_class": "MSY_H_APP",
+            "poly": [
+                ["N30.03333", "W91.43332"],
+                ["N30.02139", "W91.05837"],
+                ["N30.00003", "W90.83335"],
+                ["N29.53338", "W90.54164"],
+                ["N29.31667", "W90.54279"],
+                ["N29.31667", "W91.48333"],
+                ["N29.50002", "W91.61662"],
+                ["N29.76111", "W91.62222"],
+                ["N29.77500", "W91.52500"],
+                ["N29.90000", "W91.43333"]
+            ]
+        },
+        {
+            "floor": 0,
+            "ceiling": 110,
+            "airspace_class": "B",
+            "poly": [
+                ["N30.43111", "W90.73222"],
+                ["N30.60333", "W90.60806"],
+                ["N30.64444", "W89.79639"],
+                ["N30.46750", "W89.87778"]
+            ]
+        },
+        {
+            "floor": 0,
+            "ceiling": 40,
+            "airspace_class": "B",
+            "poly": [
+                ["N30.60332", "W90.60801"],
+                ["N30.82500", "W90.71667"],
+                ["N30.86667", "W90.64167"],
+                ["N30.86673", "W90.30001"],
+                ["N30.61667", "W90.29722"]
+            ]
+        },
+        {
+            "floor": 40,
+            "ceiling": 110,
+            "airspace_class": "B",
+            "poly": [
+                ["N30.21668", "W90.88337"],
+                ["N30.21667", "W91.05833"],
+                ["N30.02139", "W91.05837"],
+                ["N30.00003", "W90.83335"]
+            ]
+        }
+    ],
+    "fixes": {
```

Comment:
Fixes need to be alphabetised and colons aligned

Reply:
fixed